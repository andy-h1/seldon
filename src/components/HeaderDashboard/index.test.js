import React from 'react';
import { render, waitFor } from '@testing-library/react';
import { HeaderDashboard } from '.';

describe('Dashboard Header component', () => {
  it('should render', async () => {
    const { container } = render(<HeaderDashboard />);
    await waitFor(() => expect(container.firstChild).toMatchSnapshot());
  });
});
