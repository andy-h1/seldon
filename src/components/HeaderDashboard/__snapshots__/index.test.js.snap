// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Header component should render 1`] = `
<div
  class="sc-bdfBwQ jZpluA"
>
  <div>
    <div
      class="MuiTypography-root MuiTypography-h4"
    >
      Dashboard
    </div>
  </div>
  <div
    class="sc-gsTCUz dbUBDb"
  >
    <button
      aria-label="print"
      class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-1 MuiButton-containedSizeSmall MuiButton-sizeSmall"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 8h-1V3H6v5H5c-1.66 0-3 1.34-3 3v6h4v4h12v-4h4v-6c0-1.66-1.34-3-3-3zM8 5h8v3H8V5zm8 12v2H8v-4h8v2zm2-2v-2H6v2H4v-4c0-.55.45-1 1-1h14c.55 0 1 .45 1 1v4h-2z"
          />
          <circle
            cx="18"
            cy="11.5"
            r="1"
          />
        </svg>
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
    <button
      aria-label="download"
      class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-1 MuiButton-containedSizeSmall MuiButton-sizeSmall"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M13 5v6h1.17L12 13.17 9.83 11H11V5h2m2-2H9v6H5l7 7 7-7h-4V3zm4 15H5v2h14v-2z"
          />
        </svg>
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </div>
</div>
`;
